lighthouse/src/main.rs

```diff
@@ -395,10 +395,10 @@ fn main() {
             .action(ArgAction::HelpLong)
             .display_order(0)
             .help_heading(FLAG_HEADER)
+            .global(true)
```

Comment:
Does this mean we can remove the "help" arg definition in the subcommands?

Reply:
yes good point, i've gone ahead removed the "help" arg definitions in all the subcommands